### PR TITLE
BDOG-1850 add Scala 2.13 cross compile

### PR DESCRIPTION
**Pull request by Shnick**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,11 +1,19 @@
+val scala2_12 = "2.12.15"
+val scala2_13 = "2.13.7"
+
 lazy val root = (project in file("."))
-  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning)
   .settings(
     majorVersion := 5,
     name := "performance-test-runner",
     makePublicallyAvailableOnBintray := true,
```
Comment (colin-lamed) [inline build.sbt:9]: one for PR bot - this should now be `isPublicRepo`

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.14.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.13.0")
-
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.6.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.1.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.8.15")
```
Comment (colin-lamed) [inline project/plugins.sbt:8]: I don't think we should need `sbt-plugin` and `sbt-distributable`

File: .gitignore [hunk 3]
```diff
@@ -1,21 +1,39 @@
...
+.metals/
+modules
+/out
+precompiled
 /.project
+project/metals.sbt
+project/project
+project/target
 /RUNNING_PID
+server.pid
```
Comment (colin-lamed) [inline .gitignore:33]: some of these entries shouldn't apply

File: build.sbt (hunk 1 above)
Comment (Shnick) [reply build.sbt]: Thanks I've changed `makePublicallyAvailableOnBintray` to `isPublicArtefact`

File: project/plugins.sbt (hunk 2 above)
Comment (Shnick) [reply project/plugins.sbt]: Removed

File: .gitignore (hunk 3 above)
Comment (Shnick) [reply .gitignore]: Removed

**Review by vivrichards600 — APPROVED** [review]

**Pull request merged**